.devcontainer/devcontainer.json

```diff
@@ -48,9 +48,9 @@
 		8080,
 		3306
 	],
-	// Use 'postCreateCommand' to run commands after the container is created.
-	// "postCreateCommand": "sudo chmod a+x \"$(pwd)\" && sudo rm -rf /var/www/html && sudo ln -s \"$(pwd)\" /var/www/html"
 	// Set `remoteUser` to `root` to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
 	"remoteUser": "vscode",
+	// Use 'postCreateCommand' to run commands after the container is created.
```

Comment:
I think we should add a check if `vendor` exists, only let composer install at the first time.

Sometimes we need to edit the files in vendor directly, when we rebuild the container and trigger this install it would be affect this.

Reply:
`[ ! -d "vendor" ] && composer i`

Reply:
done